docs/pages/server-access/guides/bpf-session-recording.mdx

```diff
@@ -47,43 +40,29 @@ $ uname -r
 5.8.17
 ```
 
-### Operating System and Kernel Status table
-
-<Tabs>
-  <TabItem label="Ubuntu">
-    | | | Kernel Version |
-    | - | - | - |
-    | 20.10 | Groovy Gorilla | 5.8 ✅ |
-  </TabItem>
-
-  <TabItem label="Fedora">
-    | | | Kernel Version |
-    | - | - | - |
-    | 33 | 5.8 ✅ |
-  </TabItem>
-
-  <TabItem label="archlinux">
-    | | Kernel Version |
-    | - | - |
-    | 2020.09.01 | 5.8.5 ✅ |
-  </TabItem>
+### Linux distributions and supported kernels
 
-  <TabItem label="Flatcar">
-    | | Kernel Version |
-    | - | - |
-    | 2765.2.2 | 5.10.25 ✅ |
-  </TabItem>
-</Tabs>
+<table>
```

Comment:
Better to present this in table format.

Reply:
@inertial-frame to be honest, I actually prefer `<table>` vs markdown tables. I tend to spend a lot of time fixing the alignment, etc. If you think there is a lot of value in markdown format, I can tweak it. Let me know

Reply:
Should have been more clear here - this is better in HTML Table (or Markdown Table format) rather than Tabs. Totally agree!